mmv1/products/appengine/api.yaml

```diff
@@ -477,6 +477,15 @@ objects:
             required: true
             description: |
               Full Serverless VPC Access Connector name e.g. /projects/my-project/locations/us-central1/connectors/c1.
+          - !ruby/object:Api::Type::Enum
+            name: 'egressSetting'
+            required: false
+            values:
+              - :EGRESS_SETTING_UNSPECIFIED
+              - :ALL_TRAFFIC
+              - :PRIVATE_IP_RANGES
+            description: |
+              The egress setting for the connector, controlling what traffic is diverted through it.
```

Comment:
Could this value be a string instead with the potential value list linked to api documentation. We see enum fields as difficult to maintain as new values can get can add with varying frequency.